src/main/java/org/folio/dew/service/BulkEditParseService.java

```diff
@@ -92,7 +92,7 @@ private String getPatronGroupId(UserFormat userFormat) {
   private List<String> getUserDepartments(UserFormat userFormat) {
     String[] departmentNames = userFormat.getDepartments().split(ARRAY_DELIMITER);
     return Arrays.stream(departmentNames).parallel()
-      .map(name -> userReferenceService.getDepartmentByName(name))
+      .map(userReferenceService::getDepartmentByName)
```

Comment:
Empty strings should be excluded from processing

Reply:
Thank you, corrected.